compute/resource_job.go

```diff
@@ -144,6 +154,12 @@ func ResourceJob() *schema.Resource {
 			if err != nil {
 				return err
 			}
+			if js.NewCluster != nil {
+				err = validateClusterDefinition(*js.NewCluster)
+				if err != nil {
```

Comment:
i'd suggest to collapse it into one line like `if err = validateClusterDefinition(*js.NewCluster); err != nil {`